habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -225,6 +225,9 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
                 obs_k = obs_k.astype(np.uint8)
             if obs_k.shape[2] == 1:
                 obs_k = np.concatenate([obs_k for _ in range(3)], axis=2)
+            if 'third' not in sensor_name:
+                height, width = obs_k.shape[:2]
+                obs_k[height//2 - 3: height//2 + 3, width//2 - 3: width//2 + 3] = [[255, 0, 0]]
```

Comment:
Could you explain what we're doing here?

Reply:
I mark a small square at the center in red color before saving the visualization